### ISSUE TITLE
[a11y][MultiComboBox] - Multi Combo box focus is not visible with keyboard navigation in mobile mode

### ISSUE THREAD

**Issue by yoganjan** [SAP/ui5-webcomponents#8064]
### Bug Description

Multi Combo box focus is not visible when we navigate with the keyboard tabbing in mobile mode. 

### Affected Component

MultiComboBox

### Expected Behaviour

Focus should be visible to the user , when we navigate with keyboard tabbing. 

### Steps to Reproduce

1. Open https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-multicombobox--docs
2. Change to mobile mode from chorme dev tools
3. Try to navigate to MultiComobox using key board tabbing 
4. Observe that the focus is not visible in mobile. (However focus is visible in desktop verion)
...

### Priority

Medium

### UI5 Web Components Version

1.20.0

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by kgogov** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,

After examining the reported issue, I found that when opening the ui5-multi-combobox [sample](https://sap.github.io/ui5-webcomponents/playground/iframe.html?id=main-multi-combo-box--basic&viewMode=story) in the browser's dev tools and pressing the **Tab** key, the element **doesn't become active or receive focus**.

I compared this behavior to an [ui5-input](https://sap.github.io/ui5-webcomponents/playground/iframe.html?args=&id=main-input--basic&viewMode=story) or an [ui5-select](https://sap.github.io/ui5-webcomponents/playground/iframe.html?args=&id=main-select--basic&viewMode=story) components that does receive focus under similar conditions.

Could you please check this?

Best,
Konstantin

<img width="1408" alt="multicombobox-focus" src="https://github.com/SAP/ui5-webcomponents/assets/68374332/40fa4640-53fd-4015-9342-1c6392f27e54">

**Comment by niyap** [comment]
Hello @yoganjan,

On mobile device, there is no TAB key. Also, when we tap over the ui5-multi-combobox web component, we do not expected to see the input field focused, but to open the dialog with the suggestions. The component behaves like expected.

Kind Regards,
Niya

**Comment by yoganjan** [comment]
Hi @niyap , When we connect keyboard with bluetooth , we can do the tabbing. Team is doing the testing with bluetooth enabled keyboad.

**Comment by yoganjan** [comment]
Hi @niyap  , Do we need to raise a different bug or this will be re-openedas this issue is reproducible in mobile devices with Blueetooth enabled keybaord

**Comment by niyap** [comment]
Hello @yoganjan,

I'm sorry I missed your answer. I noticed that a separate issue is already created, so we will analyse the problem there.

Kind Regards,
Niya
